nagios/config/wavefront-csp-oauth.cfg

```diff
@@ -0,0 +1,45 @@
+# Nagios command definitions for host and service metrics to the Wavefront server using CSP Oauth
+define command {
+    command_name  wavefront-perf-host
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py /usr/local/nagios/var/host-perfdata \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_app_id [CSP_APP_ID] \
+                                                        --csp_app_secret [CSP_APP_SECRET] \
+                                                        --csp_base_url [CSP_BASE_URL]
+}
+
+
+define command {
+    command_name  wavefront-perf-service
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py /usr/local/nagios/var/service-perfdata \
+                                                        --service \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_app_id [CSP_APP_ID] \
+                                                        --csp_app_secret [CSP_APP_SECRET] \
+                                                        --csp_base_url [CSP_BASE_URL]
+}
+
+# Wavefront contact definition for event notification commands using CSP Oauth
```

Comment:
Operations for Applications contact definition for event notification commands using the credentials (id and secret) of a server to server OAuth app in VMware Cloud services.